quickstarts/host-inventory-quickstart.yaml

```diff
@@ -21,13 +21,31 @@ spec:
 
 
   tasks:
-    - description: >-
+    - description: |-
 
-        1.  Configure your host inventory settings. View [documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim) to follow the steps to configure your host inventory settings.
+        1.  Configure your host inventory settings.
+
+          a. From the **All Clusters** view of the OpenShift Container Platform web console, navigate to the host inventory by clicking infrastructure > **Host inventory**
+
+          b. Select **Configure host inventory settings**.
+
+        1. Enter the required values for **Database storage**, **System Storage**, and **Image Storage** in the modal that opens. 
+
+          a. Enter the required values for Database storage, System Storage, and Image Storage.
```

Comment:
Woops! looks like this step is still here after making it a task.

Reply:
Thanks @opayne1 , just changed it :)